snap/snapcraft.yaml

```diff
@@ -0,0 +1,59 @@
+name: pebble
+adopt-info: pebble-release-data
+summary: Pebble - take control of your internal daemons!
+description: |
+  Pebble helps you to orchestrate a set of local service processes
+  as an organized set. It resembles well-known tools such as supervisord,
+  runit, or s6, in that it can easily manage non-system processes
+  independently from the system services, but it was designed with unique
+  features that help with more specific use cases.
+
+  **Usage**
+
+  * The Pebble snap's default directory is
+  `$HOME/snap/pebble/<rev>/var/lib/pebble/default`. This directory is **not**
+  created at installation time.
+
+  **Documentation**
+
+  To learn more about Pebble please check the project's documentation at
+  https://github.com/canonical/pebble.
+issues: https://github.com/canonical/pebble/issues
+source-code: https://github.com/canonical/pebble
+license: GPL-3.0
+base: bare
+build-base: core22
+confinement: classic
+
+apps:
+  pebble:
+    command: pebble-wrapper
+
+parts:
+  pebble:
+    plugin: go
+    source: .
+    build-snaps:
+      - go
+    build-environment:
+      - CGO_ENABLED: 0
+      - GOFLAGS: -ldflags=-w -ldflags=-s
+
+  pebble-release-data:
+    plugin: nil
+    source: .
+    override-build: |
+      version="$(cat cmd/VERSION || ./cmd/mkversion.sh --output-only)"
```

Comment:
I presume the built Pebble binary is not yet available? If so we could use `pebble version --client`.

Reply:
Yes we can. That's better indeed. Thanks. Applied in https://github.com/canonical/pebble/pull/179/commits/c21e9c5cbad88164f677f9923190f2e7efe1c573